en/advanced/ocr-engine-selection.md

```diff
@@ -0,0 +1,153 @@
+# ADR-002: OCR Engine Selection for JabRef
+
+**Status:** proposed
+**Date:** 2025-07-13
+**Decision-makers:** JabRef development team, GSoC contributor (Kaan Erdem)
+**Consulted:** Academic users, library scientists, OCR technology experts
+**Informed:** JabRef community, contributors, institutional users
+
+## OCR Engine Selection for Academic Document Processing
+
+### Context and Problem Statement
+
+JabRef requires an OCR engine to extract text from scanned PDFs and image-based academic documents. Tesseract is currently implemented, but accuracy varies significantly with document quality and type. Academic documents present unique challenges: mathematical notation, multiple languages, complex layouts, tables, and mixed handwritten/printed content. Which OCR engine(s) should JabRef adopt to best serve its academic user base while balancing accuracy, cost, privacy, and implementation complexity?
+
+### Decision Drivers
+
+* Accuracy on academic document types (printed papers, scanned books, handwritten notes)
+* Privacy requirements for unpublished research materials
+* Cost constraints (open-source project with limited funding)
+* Language support for international academic community
+* Support for mathematical and scientific notation
+* Offline capability for secure research environments
+* Processing speed for batch operations
+* Implementation and maintenance complexity
+* Table and structure extraction capabilities
+* Long-term sustainability and community support
+
+### Considered Options
+
+* Option 1: Tesseract OCR (current implementation)
+* Option 2: Google Cloud Vision API
+* Option 3: AWS Textract
+* Option 4: Microsoft Azure Computer Vision
+* Option 5: EasyOCR
+* Option 6: PaddleOCR
+* Option 7: ABBYY FineReader SDK
+
+### Decision Outcome
+
+Chosen option: "Option 1: Tesseract OCR", with planned addition of "Option 2: Google Cloud Vision API" as an optional premium feature, because Tesseract provides a solid free foundation while Google Vision offers superior accuracy for users willing to trade privacy for performance.
+
+### Consequences
+
+* **Good**, because maintains free, privacy-preserving option as default
+* **Good**, because allows users to opt-in to higher accuracy when needed
+* **Good**, because Tesseract's 100+ language support covers academic needs
+* **Good**, because implementation is already complete and tested
+* **Bad**, because Tesseract struggles with handwritten text
+* **Bad**, because requires additional development for cloud integration
+* **Bad**, because increases support complexity with multiple engines
+
+### Confirmation
+
```

Comment:
Elaborate on how this is done. I would assume that you have the 100+ PDFs at hand and wrote a test suite?

Reply:
An ADR can also have TODOs and links to existing drafts of the test suite.